sketch.js

```diff
@@ -38,9 +45,9 @@ function draw() {
   // second copy of the image right next to it
   // once the second image gets to the 0 point, we can reset bgX to
   // 0 and go back to drawing just one image.
-  if(bgX <= -bgImg.width + width){
+  if (bgX <= -bgImg.width + width) {
```

Comment:
yay spaces! 😍

Reply:
I have personally never understood spaces after `if`/`for`/`when` - the brackets are a part of the keyword construct so why would they be separated? It would be like having function calls as `loadImage ('...');`, which, although syntactically valid, I have never seen anyone use.

(Note this isn't a request to change, I'm just wondering what the reasoning behind this is)

Reply:
Haha, interesting point! In my mind the space actually *distinguishes* the structure from a function call -- `if`, `for`, and `while` being special control structures? But this may just be what I tell myself to sleep at night as the missing space sends me into a panicked spiral.

Reply:
In my reasoning, a branch construct has three or more parts, which are space separated: the keyword, the condition and the code-block. Additionally an else/elseif can be added. For me this sounds like reason to add spacing.

Reply:
Ah, interesting. Those both sound like good reasoning. I think I'd be more likely to also be for that space convention if the form of `if a b else c` was allowed without the brackets (See Rust for an example when I think the syntax feels very intuitive). With them as required it feels to me like they should be attached, but thanks for telling me your thoughts!